lib/benchee/conversion/memory.ex

```diff
@@ -46,6 +46,131 @@ defmodule Benchee.Conversion.Memory do
               }
   }
 
+  # Unit conversion functions
+
+  @doc """
+  Converts a value for a specified unit and converts it to the equivalent of another unit of measure.
+
+  ## Examples
+
+    iex> {value, unit} = Benchee.Conversion.Memory.convert({1024, %Benchee.Conversion.Unit{name: :kilobyte}}, %Benchee.Conversion.Unit{name: :megabyte})
+    iex> value
+    1.0
+    iex> unit.name
+    :megabyte
+  """
+  def convert({value, current_unit}, desired_unit) when current_unit == desired_unit do
```

Comment:
I think it would be helpful to add a typespec here. If you're unfamiliar with typespecs, here's some more information: https://elixir-lang.org/getting-started/typespecs-and-behaviours.html, and you can find other examples in many other places in the codebase here. And just holler if you have other questions!

Reply:
If type spec seems too much, please let us know and we could fill in

Reply:
I'll add some typespecs with the next commit. Thanks for the doc link!